docs/undergrad.yaml

```
@@ -438,8 +438,8 @@ Topology:
     example, the Hilbert basis of trigonometric polynomials: 'fourier_Lp'
     its orthonormality: 'orthonormal_fourier'
     its completeness: 'span_fourier_Lp_closure_eq_top'
-    example, classical Hilbert bases of orthogonal polynomials: ''
-    Lax-Milgram theorem: ''
+    example, classical Hilbert bases of orthogonal polynomials, their orthonormality and completeness: ''
```

```suggestion
    example, classical Hilbert bases of orthogonal polynomials: ''
```
This probably got caught in the merge ... I rephrased it last week now that Hilbert bases have the orthonormality and completeness bundled in.